codecov.yml

```diff
@@ -0,0 +1,46 @@
+# Enable coverage report message for diff on commit
+# Docs can be found here: https://docs.codecov.io/v4.3.0/docs/commit-status
+coverage:
+  status:
+    project:
+      tests:
+        target: 100%
```

Comment:
Sounds like a pretty high target :-)

Reply:
We'll see how it goes, but generally, I think all test code should actually run

Reply:
Ah got it, I misread that config option. 👍